src/plugins/condition/components/ConditionItem.vue

```diff
@@ -101,6 +107,7 @@
             <input
               v-if="selectedOutputSelection === outputOptions[2]"
               v-model="condition.configuration.output"
+              aria-label="Condition Output String Input"
```

Comment:
No need to include `Input` here as the screen reader will read out the type of the element before the label.